addons/token-exchange/manifests/hub_clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: open-cluster-management:token-exchange-agent:admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  kind: User
+  apiGroup: rbac.authorization.k8s.io
+  name: {{ .User }}
```

Comment:
also use group here

Reply:
you mean add user and group?

I still don't understand how adding RBAC to a user on hub allows me to fetch resources on Spoke. Do you have a theory? Am I doing something wrong in my permissionConfig function?

Reply:
this  one is not the permission to fetch  resource on spoke, right? it is for agent to access to hub so it  can sync  the secret to hub.

Reply:
It's actually required to allow agent to watch secrets on spoke.

Reply:
since the file is put here https://github.com/red-hat-storage/odf-multicluster-orchestrator/pull/8/files#diff-3e25c4523000c43768d352c8ab7c105eecca179fa2043867e7050507783a141fR46 . It means you are apply this on the hub which allows the  agent on  spoke to access any secret on hub.

Reply:
So even without these role/rolebinding on hub, agent on spoke should still be able to access secret on spoke right?

Reply:
right